src/gov-action-contracts/TimelockRolesUpgrader.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.26;
+
+import {TimelockControllerUpgradeable} from "openzeppelin-upgradeable/governance/TimelockControllerUpgradeable.sol";
+
+contract TimelockRolesUpgrader {
```

Comment:
Let's figure out whether we need to use the address registry here per Arbitrum docs